src/controllers/shopController.ts

```diff
@@ -8,32 +7,27 @@ export const createShop = async (req: Request, res: Response) => {
     shopDescription,
```

Comment:
Maybe we can put these types into a separate file, because they might be useful when we transfer data from frontend to backend! (frontend can pass in their info as these types)